code/modules/mob/living/silicon/robot/robot.dm

```diff
@@ -165,6 +165,11 @@ GLOBAL_LIST_INIT(robot_verbs_default, list(
 	..()
 
 	add_robot_verbs()
+	
+	// Remove inherited verbs that effectively do nothing for cyborgs, or lead to unintended behaviour.
+	verbs -= /mob/living/verb/lay_down
+	verbs -= /mob/living/verb/mob_sleep
+	verbs -= /mob/living/verb/resist
```

Comment:
If someone goes from being a borg to a human, do they get these back?
(Remove MMI, remove brain, put in humanised monkey, etc)

Reply:
Yep